Web/starter_project_barbies/components/Home-page/AcceptanceCard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { AcceptanceRate } from '../../interface/interfaces'
+function AcceptanceCard({data}:{data:AcceptanceRate}) {
```

Comment:
why the need to take it as data when it is actually an `acceptanceRate` (as you have specified in the type too)?

Reply:
i suggest you destruct it and use it something like this `(acceptanceRate: {year, passed, average}}) => {...`
that should help you avoid doing `acceptanceRate.` everytime